docs/computing/alliancecan.md

```diff
@@ -156,6 +171,59 @@ ssh -Y <user>@graham.computecanada.ca
 By default, this brings you to the login node which **is not** a compute node. 
 This should only be used for viewing images.
 
+### Run Jupyter Notebooks in your local VS Code
+To be able to run Jupyter Notebooks in Visual Studio Code from your local machine, use the following instructions:
+1. Follow the instructions above to set the passwordless SSH.
```

Comment:
I would make this a call out - either {: .note} or {: .important} so that it stands out and the user knows this needs to be set beforehand.